account_financial_report_webkit/models/account_static_balance.py

```diff
@@ -209,15 +277,16 @@ def map_data_to_account_id(data):
         static_periods = periods.filtered(
             lambda p: p.state == 'done' and not p.special)
         if static_periods:
-            static_periods -= self.get_missing_periods_and_accounts(
+            static_periods -= self.get_missing_periods_accounts_and_journals(
```

Comment:
The whole idea of the exercise was that when computing the periods per journal, we could benefit from balances for closed journals of that period so that we would only have to compute the balances for some journals of that period, hoping that that would give us some performance gain.
It looks like you did not implement that, because you only recompute full periods here. Am I missing something?

Reply:
I implemented by reopening the journal periods that only the specific journal period will be unlinked from the static balance (see: https://bitbucket.org/emagine-reality/essentbe-odoo/pull-requests/2125 ).

Reply:
Extended test_04 ( https://github.com/EssentNovaTeam/account-financial-reporting/pull/6/files#diff-efe18539c3d24a5b0cb5da09cc2c1a95dbac0a5642565fad6eaea5bf7ca9e984R96 ) to validate that the balances are only generated for given journal periods

Reply:
But I don't see you use the balances of the remaining closed periods here.